data/extract_rosbag_to_tf.py

```diff
@@ -220,6 +238,8 @@ def main():
                 else:
                     first_left_image_time = time
                     left_event_image_times.append(time.to_sec())
+                    # filter events we added previously
+                    left_events = filter_events(left_events, left_event_image_times[-1] - t_start)
```

Comment:
Is this necessary? My understanding was that this was handled in _save_events (although I agree that this is a much cleaner way to do this).

Reply:
I'm not sure that this is handled in _save_events. For me it was not so obvious, thus I added this check.

Reply:
I see. It should be implicitly handled in _save_events here: https://github.com/alexzzhu/EV-FlowNet/blob/3c2bcb3c626e5bc2916d14f836d1528400b6d793/data/extract_rosbag_to_tf.py#L70, due to the way lists are passed as arguments in python.

However, I agree that this provides an explicit guarantee, and may catch some missed events.